examples/gte/models/gte_embedding.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass
+import torch
+from nos.hub import HuggingFaceHubConfig
+from sentence_transformers import SentenceTransformer, util
+import numpy as np
+from typing import Dict, List, Union
+
+@dataclass(frozen=True)
+class GTEEmbeddingConfig(HuggingFaceHubConfig):
+    """Sentence Embedding model configuration."""
+
+    max_input_token_length: int = 512
+    """Maximum number of tokens in the input."""
+
+    compute_dtype: str = "float16"
```

Comment:
nit You can probably remove this, since it's not being used in the model `__init__`.

Reply:
Yeah, Will do. I think I put this here for future use with models of different context sizes. Can re-implement later